scripts/project-depcheck.ts

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ts-node
+
+import path from 'path'
+
+import depcheck from 'depcheck'
+
+const options = {
+  detectors: [
+    // the target detectors
+    depcheck.detector.requireCallExpression,
+    depcheck.detector.importDeclaration,
+  ],
+  ignoreBinPackage: false, // ignore the packages with bin entry
+  ignoreDirs: [
+    // folder with these names will be ignored
+    'public',
+    '.cache',
+    'static',
+  ],
+  // ignore dependencies that matches these globs
+  ignoreMatches: [],
+  parsers: {
+    // the target parsers
+    '*.ts': depcheck.parser.typescript,
+    '*.tsx': depcheck.parser.typescript,
+  },
+  skipMissing: false, // skip calculation of missing dependencies
+  specials: [
+    // the target special parsers
+    depcheck.special.eslint,
+    depcheck.special.babel,
+    depcheck.special.tslint,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)